Monofoxe/Monofoxe.Engine/HsvColor.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Monofoxe.Engine
+{
+  public struct HsvColor
+  {
+		public float _hue;
```

Comment:
You got all the variables mixed up. Reorganize them, so that it would look like
```
public PublicProperty {get; set;}
private _privateProperty;
```

Also make them private.